packages/common/src/entities/dbmss/dbmss.abstract.ts

```diff
@@ -142,87 +124,75 @@ export abstract class DbmssAbstract<Env extends EnvironmentAbstract> {
     /**
      * @hidden
      */
-    runQuery<Res = any>(
-        driver: Driver<TapestryJSONResponse<Res>>,
+    /* eslint-disable-next-line require-await */
+    async runReadQuery(
```

Comment:
You don't need the async here if you're returning the promise you get from the driver.
```suggestion
    runReadQuery(
```